src/test/java/gov/healthit/chpl/aqa/asserts/ProductHistoryEyeAsserts.java

```diff
@@ -0,0 +1,45 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.testng.Assert.assertTrue;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.ProductHistoryEyePage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class ProductHistoryEyeAsserts definition.
+ */
+public class ProductHistoryEyeAsserts extends Base {
+
+    /**
+     * Assert that expected developer edit details is displayed on product history eye.
+     * @param expectedDeveloperEditDetails is "Developer changed from"
+     */
+    @Then("^I see the product history of developer edit is recorded as \"([^\"]*)\"$")
+    public void iSeeProductHistoryOfDeveloper(final String expectedDeveloperEditDetails) {
+        String actualDeveloperEditDetails = ProductHistoryEyePage.productHistoryDeveloper(getDriver()).getText();
+        assertTrue((actualDeveloperEditDetails.startsWith(expectedDeveloperEditDetails)));
```

Comment:
I guess I'm not sure about this logic... you're looking up the element in ProductHistoryEyePage by looking for the text "Developer changed from". But then you are asserting that the elements text starts with "Developer changed from". This seems like a duplicate check to me. If the ProductHistoryEyePage.productHistoryDeveloper() method returns an element at all then I think you know that the edit history is there. It's not really wrong but just seems like you don't need to check twice.